validator/src/validator.rs

```diff
@@ -50,6 +49,51 @@ pub fn run_validator(config: Config) {
         .expect("Invalid Genesis Hash");
 
     loop {
+        // Check if there is a pending block to sign
+        let chain_account: ChainData = program.account(chain).unwrap();
+        if chain_account.pending_block().unwrap().is_some() {
+            if chain_account
+                .pending_block()
+                .unwrap()
+                .unwrap()
+                .signers
+                .get(&validator.pubkey().into())
+                .is_none()
+            {
+                log::info!("it has a pending block");
+                let fingerprint = &chain_account
+                    .pending_block()
+                    .unwrap()
+                    .unwrap()
+                    .fingerprint;
+                let signature = validator.sign_message(fingerprint.as_slice());
+
+                let tx = utils::submit_call(
+                    &program,
+                    signature,
+                    fingerprint.as_slice(),
+                    &validator,
+                    chain,
+                    trie,
+                );
+                match tx {
+                    Ok(tx) => {
+                        log::info!(
+                            "Pending Block signed -> Transaction: {}",
+                            tx
+                        );
+                        break;
+                    }
+                    Err(err) => {
+                        log::error!("Failed to send the transaction {err}")
+                    }
+                }
+            } else {
+                log::info!("You have already signed the pending block");
```

Comment:
Don't log saying "you.."
"There are ..."

Reply:
oh will do